ccd-definition/FixedLists/HearingVenue.json

```diff
@@ -0,0 +1,618 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingVenue",
+    "ListElementCode": "1",
```

Comment:
Not sure I am a fan of listElementCode: 1 etc 🤔 

Are we fine with the displayed values all being caps?